pkg/utils/tekton/pipelines.go

```diff
@@ -87,6 +81,7 @@ type VerifyEnterpriseContract struct {
 	EffectiveTime       string
 }
 
+// Generates pipelineRun from VerifyEnterpriseContract.
 func (p VerifyEnterpriseContract) Generate() *v1beta1.PipelineRun {
```

Comment:
since this method's working with PipelineRun object, should it live in `pipelinerun.go` instead

Reply:
Moved both functions for generating pipelineRuns to `pipelineruns.go`.